File: test/fixtures/output/ocaml/cohttp/multipart-file.ml
open Cohttp_lwt_unix
open Lwt

let uri = Uri.of_string "http://mockbin.com/har" in
let headers = Header.init ()
  |> fun h -> Header.add h "Content-Type" "multipart/form-data"
in
let body = "foo=Hello World" in

Client.call ~headers ~body (Code.method_of_string "POST") uri
>>= fun (res, body_stream) ->
  (* Do stuff with the result *)

File: test/fixtures/output/javascript/fetch/multipart-file.js
const form = new FormData();
form.append('foo', 'test/fixtures/files/hello.txt');

const options = {
  method: 'POST',
  headers: { 'content-type': 'multipart/form-data; boundary=---011000010111000001101001' },
};

options.body = form;

fetch('http://mockbin.com/har', options)
  .then(response => response.json())
  .then(response => console.log(response))
  .catch(err => console.error(err));
